### PR TITLE
standardize repo publishing

### PR DESCRIPTION
**Pull request by wsbrunson**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2016 MyCompany
```
Comment (gregjopa) [inline LICENSE]:
```suggestion
   Copyright 2017 PayPal
```

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
```
Comment (gregjopa) [inline LICENSE]:
```suggestion
```

I think this Appendix and the paragraph below can be deleted. It's just directions for the person using the license.

**Review by gregjopa — APPROVED** [review]
💯

**Comment by westeezy** [comment]
Can you also remove the `publish.sh` file?

**Review by westeezy — APPROVED** [review]

**Pull request merged**